app_dart/test/request_handlers/luci_status_test.dart

```diff
@@ -47,7 +47,7 @@ void main() {
 
   setUp(() async {
     serviceAccountInfo = const ServiceAccountInfo(email: serviceAccountEmail);
-    config = FakeConfig(luciTryInfraFailureRetriesValue: 2, deviceLabServiceAccountValue: serviceAccountInfo);
+    config = FakeConfig(deviceLabServiceAccountValue: serviceAccountInfo);
```

Comment:
Aren't we testing the retryCount anymore?

Reply:
We are. [`maxLuciTaskRetries`](https://devops-console-oss.corp.google.com/flutter/infra/+/master:config/devicelab_config.star) is the one we are using. `luciTryInfraFailureRetriesV` is some legacy parameter we have deprecated.